packages/transformers/xml/src/processing-instruction.js

```diff
@@ -0,0 +1,13 @@
+// @flow
+import type {MutableAsset} from '@parcel/types';
+
+// FIXME: Parse per https://www.w3.org/TR/xml-stylesheet/#pseudo-attributes
+const HREF_PATTERN = /(?<=\bhref[ \t\n]*=[ \t\n]*(['"])).*?(?=\1)/;
```

Comment:
Technically the processing instruction contains [pseudo-attributes](https://www.w3.org/TR/xml-stylesheet/#pseudo-attributes) that should be parsed similarly to XML attributes. Perhaps there’s a way to leverage [xmldom](https://github.com/xmldom/xmldom) to do this more robustly?